src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2013,13 +2013,14 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
 
     if (human)
     {
-        lastClose[jss::converge_time_s] = static_cast<double> (
-            mConsensus->getLastCloseDuration()) / 1000.0;
+        using dsec = std::chrono::duration<double>;
```

Comment:
Food for thought: maybe we should avoid having a `using` for such one-off uses.

    lastClose[jss::converge_time_s] =
        std::chrono::duration<double> (
            mConsensus->getLastCloseDuration()).count();